smart-contracts/contracts/Unlock.sol

```diff
@@ -86,7 +86,8 @@ contract Unlock is IUnlock, Initializable, Ownable {
   function createLock(
     uint _expirationDuration,
     uint _keyPrice,
-    uint _maxNumberOfKeys
+    uint _maxNumberOfKeys,
+    uint _cancelRefundPenaltyDenominator
```

Comment:
Same comment as above? I think it is fine to set the default `cancelRefundPenaltyDenominator` and then have a setter for it so that we do not need to change the signature of `createLock` everywhere?